src/privatesend-server.cpp

```diff
@@ -280,6 +280,7 @@ void CPrivateSendServer::SetNull()
 {
     // MN side
     vecSessionCollaterals.clear();
+    nSessionMaxParticipants = NULL;
```

Comment:
@PastaPastaPasta Let's try and avoid using NULL in the future and follow best practices. For integers, 0 should be used and for pointers, nullptr.